docker-compose.yml

```diff
@@ -6,16 +6,19 @@ services:
 
   server:
     extends: base
-    command: npm start
+    command: gulp watch-server
     environment:
       - "MONGO_ADDR=mongodb://mongo:27017/rope"
+      - "WEB_URL=http://localhost:9001"
     ports:
       - "9000:9000"
     links:
       - mongo
     volumes:
-      - ./app:/src/app
-      - ./server:/src/server
+      - ./app:/opt/rope/app
+      - ./server:/opt/rope/server
+    depends_on:
+      - mongo
```

Comment:
@cihangir even though i set this here, the server still runs before the `mongo` container on the initial `docker-compose up`, should we use a script to wait for mongo to be available? if so can you point me to an example?

Reply:
There is [this](https://github.com/Eficode/wait-for) script which is used widely.

You can see the example here ```command: sh -c './wait-for db:5432 -- npm start```